docs/notebooks/qhsri-tutorial.pct.py

```diff
@@ -0,0 +1,232 @@
+# ---
+# jupyter:
+#   jupytext:
+#     text_representation:
+#       extension: .py
+#       format_name: percent
+#       format_version: '1.3'
+#       jupytext_version: 1.14.1
+#   kernelspec:
+#     display_name: Python 3 (ipykernel)
+#     language: python
+#     name: python3
+# ---
+
+# %% [markdown]
+# # Batch HSRI Tutorial
+
+# %% [markdown]
+# Batch Hypervolume Sharpe Ratio Indicator (qHSRI) is a method for picking a batch of query points during Bayesian Optimisation. It makes use of the Sharpe Ratio, a portfolio selection method used in investment selection to carefully balance risk and reward.
+#
+# This tutorial will cover the main points of how this algorithm works, and then demonstrate how to use the `trieste.acquisition.rule.BatchHypervolumeRatioIndicator` acquisition rule.
+#
+#
+# First we will set up our problem and get our initial datapoints. For this walkthrough we will use the noiseless scaled Branin objective.
+
+# %%
+
+import tensorflow as tf
+import matplotlib.pyplot as plt
+
+from trieste.objectives import ScaledBranin
+from trieste.objectives.utils import mk_observer
+from trieste.space import Box
+
+tf.random.set_seed(1)
+
+# Create the observer
+observer = mk_observer(ScaledBranin.objective)
+
+# Define Search space
+search_space = Box([0, 0], [1, 1])
+
+# Set initial number of query points
+num_initial_points = 5
+initial_query_points = search_space.sample(num_initial_points)
+initial_data = observer(initial_query_points)
+
+# %% [markdown]
+# We can now fit a GP to our initial data.
+
+# %%
+from trieste.models.gpflow import GaussianProcessRegression, build_gpr
+
+# Set up model
+gpflow_model = build_gpr(initial_data, search_space, likelihood_variance=1e-7)
+model = GaussianProcessRegression(gpflow_model)
+
+# %% [markdown]
+# Now consider how we might want to select a batch of $q$ query points to observe. It would be useful for some of these to be "safe bets" that we think are very likely to provide an improvement. It would also be valuable for some of these to be sampling parts of the space that we have no idea what the expected observed value is (i.e. high variance). This problem is very similar to that encountered in building finance portfolios, where you want a mix of high  risk/high reward and low risk/low reward assets. You would also want to know how much of your total capital to invest in each asset.
```

Comment:
perhaps weird spacing in **high risk/high reward and low risk/low reward assets**